build_extra/toolchain/win/mods.gni

```diff
@@ -20,9 +20,9 @@ import("//build/toolchain/cc_wrapper.gni")
 # Some python scripts that are run by BUILD.gni live here.
 base_toolchain_dir = "//build/toolchain/win"
 
-# If cc_wrapper if is set, wrap it in quotes and add a space to it.
+# If cc_wrapper if is set, add a space to it.
 if (cc_wrapper == "") {
   cc_wrapper_prefix = ""
 } else {
-  cc_wrapper_prefix = "\"$cc_wrapper\" "
+  cc_wrapper_prefix = "$cc_wrapper "
```

Comment:
Is this something that should be sent upstream?

Reply:
Not this particular change, but `build_extra/toolchain/win/*`: yes.